solutions/drf/restaurants/migrations/0001_initial.py

```diff
@@ -0,0 +1,24 @@
+# Generated by Django 3.2.15 on 2023-01-18 14:46
```

Comment:
Please, check the Django version you are using and pin the right one in the requirements.txt

"sports" boilerplate was generated by  Django 4.1.5 on 2023-01-15 17:38

Reply:
ahh yes, I used different virtual envs.